x/staking/keeper/msg_server.go

```diff
@@ -106,7 +106,7 @@ func (k msgServer) CreateValidator(ctx context.Context, msg *types.MsgCreateVali
 
 	commission := types.NewCommissionWithTime(
 		msg.Commission.Rate, msg.Commission.MaxRate,
-		msg.Commission.MaxChangeRate, sdkCtx.BlockHeader().Time,
+		msg.Commission.MaxChangeRate, sdkCtx.HeaderInfo().Time,
```

Comment:
Wow, these can of changes definitely need an UPGRADING.md section (Unreleased > Modules > All)

Would it be bad to make BlockHeader() getter quite alias HeaderInfo(), so that if a dev forgets it, it does not immediately give unattended behavior? (quick thought, I haven't checked the implication of that).

Reply:
this isnt required to be changed as part of this pr. I migrated to be consistent throughout the repo

Reply:
You are right, the getter is an alias to the headerinfo, commented before seing it 😅
This does make my other comment even more relevant however.